app/components/button/button.css

```diff
@@ -0,0 +1,56 @@
+.base-button {
+  padding: 8px 16px;
+  height: 40px;
```

Comment:
if we've got padding and font size, do we need height? feels a little over-constrained

Reply:
I wanted to make the height large enough so that it's easy to click the button, so I think `min-height` makes more sense here. Done

Reply:
I think in that case you could just increase vertical padding

Reply:
I'd rather use `min-height` because it doesn't depend on the font size.